.buildkite/scripts/build_kibana.sh

```diff
@@ -22,27 +22,42 @@ node scripts/build "${BUILD_ARGS[@]}"
 if is_pr_with_label "ci:build-cloud-image"; then
   echo "$KIBANA_DOCKER_PASSWORD" | docker login -u "$KIBANA_DOCKER_USERNAME" --password-stdin docker.elastic.co
   node scripts/build \
-  --skip-initialize \
-  --skip-generic-folders \
-  --skip-platform-folders \
-  --skip-cdn-assets \
-  --skip-archives \
-  --docker-images \
-  --docker-tag-qualifier="$GIT_COMMIT" \
-  --docker-push \
-  --skip-docker-ubi \
-  --skip-docker-ubuntu \
-  --skip-docker-serverless \
-  --skip-docker-contexts
+    --skip-initialize \
+    --skip-generic-folders \
+    --skip-platform-folders \
+    --skip-cdn-assets \
+    --skip-archives \
+    --docker-images \
+    --docker-tag-qualifier="$GIT_COMMIT" \
+    --docker-push \
+    --skip-docker-ubi \
+    --skip-docker-ubuntu \
+    --skip-docker-serverless \
+    --skip-docker-contexts
   docker logout docker.elastic.co
 
   CLOUD_IMAGE=$(docker images --format "{{.Repository}}:{{.Tag}}" docker.elastic.co/kibana-ci/kibana-cloud)
-  cat << EOF | buildkite-agent annotate --style "info" --context kibana-cloud-image
+  cat <<EOF | buildkite-agent annotate --style "info" --context kibana-cloud-image
 
   Kibana cloud image: \`$CLOUD_IMAGE\`
 EOF
 fi
 
+if is_pr_with_label "ci:build-fips-images"; then
```

Comment:
I think we should move this and the cloud image build into a new step after this PR, so they do not delay CI from time increase on Build Kibana Distribution and Plugins step.